### ISSUE TITLE
UserSettings with bool crashes when bool value is changed

### ISSUE THREAD

**Issue by kosl** [jedie/cli-base-utilities#50]
If UserSettings contains bool value and when changed to non-default then (de)serialisation fails with
AttributeError: 'bool' object has no attribute 'unwrap'.

**Comment by kosl** [comment]
~~~
 File "cli_base/toml_settings/api.py", line 81, in get_user_settings
    document_changed = toml2dataclass(document=user_settings_doc, instance=self.settings_dataclass)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "cli_base/toml_settings/deserialize.py", line 36, in toml2dataclass
    changed = toml2dataclass(document=sub_doc, instance=field_value, _changed=_changed)
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "cli_base/toml_settings/deserialize.py", line 70, in toml2dataclass
    elif not isinstance(field_value, type(doc_value.unwrap())):                
                                          ^^^^^^^^^^^^^^^^                                                                                                    
AttributeError: 'bool' object has no attribute 'unwrap'                                                                                                       
~~~

**Comment by jedie** [comment]
fixed by https://github.com/jedie/cli-base-utilities/pull/52

**Comment by jedie** [comment]
released in: https://pypi.org/project/cli-base-utilities/0.10.2/

**Comment by kosl** [comment]
Still toggling the value is inert.

**Comment by jedie** [comment]
Hm. Sad. We have to fix that. Currently i have no project that used a boolean value ;)

**Comment by kosl** [comment]
Well, bool is not a showstopper as it can be mimicked by int 0/1.

**Comment by jedie** [comment]
It's now fixed in 0.10.3: https://pypi.org/project/cli-base-utilities/0.10.3/
